guides/source/6_0_release_notes.md

```diff
@@ -211,10 +211,240 @@ Please refer to the [Changelog][active-record] for detailed changes.
+    ([Pull Request](https://github.com/rails/rails/pull/34906))
+
+*   Make `ROW_FORMAT=DYNAMIC` a default create table option for MySQL.
+    ([Pull Request](https://github.com/rails/rails/pull/34742))
+
+*   Add the ability to disable scopes generated by `ActiveRecord.enum`.
+    ([Pull Request](https://github.com/rails/rails/pull/34605/files))
+
+*   Make implicit ordering configurable for a column.
+    ([Pull Request](https://github.com/rails/rails/pull/34480))
+
+*   Bump the minimum PostgreSQL version to 9.3, dropping support for 9.1 and 9.2.
+    ([Pull Request](https://github.com/rails/rails/pull/34520))
+
+*   Make the values of an enum frozen, raising an error when attempting to modify them.
+    ([Pull Request](https://github.com/rails/rails/pull/34517))
+
+*   Make the SQL of `ActiveRecord::StatementInvalid` errors its own error property
+    and include SQL binds as a separate error property.
+    ([Pull Request](https://github.com/rails/rails/pull/34468))
+
+*   Add an `:if_not_exists` option to `create_table`.
+    ([Pull Request](https://github.com/rails/rails/pull/31382))
+
+*   Add support for multiple databases to `rails db:schema:cache:dump`
+    and `rails db:schema:cache:clear`.
+    ([Pull Request](https://github.com/rails/rails/pull/34181))
+
+*   Add support for hash and url configs in database hash of `ActiveRecord::Base.connected_to`.
+    ([Pull Request](https://github.com/rails/rails/pull/34196))
+
+*   Add support for default expressions and expression indexes for MySQL.
+    ([Pull Request](https://github.com/rails/rails/pull/34307))
+
+*   Add an `index` option for `change_table` migration helpers.
+    ([Pull Request](https://github.com/rails/rails/pull/23593))
+
+*   Fix `transaction` reverting for migrations. Previously, commands inside of a `transaction`
+    in a reverted migration ran uninverted. This change fixes that.
+    ([Pull Request](https://github.com/rails/rails/pull/31604))
+
+*   Allow `ActiveRecord::Base.configurations=` to be set with a symbolized hash.
+    ([Pull Request](https://github.com/rails/rails/pull/33968))
+
+*   Fix the counter cache to only update if the record is actually saved.
+    ([Pull Request](https://github.com/rails/rails/pull/33913))
+
+*   Add expression indexes support for the SQLite adapter.
+    ([Pull Request](https://github.com/rails/rails/pull/33874))
+
+*   Allow subclasses to redefine autosave callbacks for associated records.
+    ([Pull Request](https://github.com/rails/rails/pull/33378))
+
+*   Bump the minimum MySQL version to 5.5.8.
+    ([Pull Request](https://github.com/rails/rails/pull/33853))
+
+*   Use the utf8mb4 character set by default in MySQL.
+    ([Pull Request](https://github.com/rails/rails/pull/33608))
+
+*   Add the ability to filter out sensitive data in `#inspect`
+    (Pull Requests: [One](https://github.com/rails/rails/pull/33756) & [Two](https://github.com/rails/rails/pull/34208))
```

Comment:
We are following this style for multi PRs https://monosnap.com/file/N7mfqH7seXlyzhbru5JX4moxRtLan2